examples/pcap_to_parquet.py

```diff
@@ -20,32 +20,51 @@ def ip_to_integer(s):
     return struct.unpack("!I", socket.inet_aton(s))[0]
 
 
+def get_ip_protocol(s):
+    if "tcp" in s:
+        return "tcp"
+    if "UDP" in s:
+        return "udp"
+    if "EIGRP" in s:
+        return "eigrp"
+    if "ICMP" in s:
+        return "icmp"
+    return None
+
```

Comment:
I assume that this is a complex if statement because s contains other characters some of the time?  If not a direct lookup table (dictionary) would be both faster and clearer.